sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1609,6 +1609,19 @@ object SQLConf {
     .checkValues(StorageLevelMapper.values.map(_.name()).toSet)
     .createWithDefault(StorageLevelMapper.MEMORY_AND_DISK.name())
 
+  val DATAFRAME_CACHE_LOG_LEVEL = buildConf("spark.sql.dataframeCache.logLevel")
```

Comment:
Will we need to debug `cache table` as well? Shall we rename the config as 
`spark.sql.cache.logLevel`

Reply:
Also, let's make it an internal conf since it is for developers.

Reply:
I kept the Dataframe cache naming to differentiate it from the RDD cache.

Reply:
RDD is a Spark core concept. Anyway I respect your choice here.